cmsEditArticle.php

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+
+<?php
+
+$db = new PDO('mysql:host=127.0.0.1; dbname=karimPortfolioCMS', 'root');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+require_once 'dropdownDB.php';
+require_once 'getArticleImgDB.php';
+require_once 'updateArticleDB.php';
+
+$dropdownKey = 'title';
+$dropdownArray = getArticleTitles($db, $dropdownKey);
+$titleDropdown = getValuesfromDB($dropdownArray, $dropdownKey);
+
+$actionType = $_POST['updateArticle'];
```

Comment:
why all the variables? just use the $_POST variable

Reply:
removed unnecessary variables